avm/res/batch/batch-account/tests/e2e/waf-aligned/main.test.bicep

```
@@ -0,0 +1,109 @@
+targetScope = 'subscription'
+
+metadata name = 'WAF-aligned'
+metadata description = 'This instance deploys the module in alignment with the best-pratices of the Well-Architectured-Framework.'
```

Ah this typo! Any automation to be updated?

```suggestion
metadata description = 'This instance deploys the module in alignment with the best-practices of the Azure Well-Architected Framework.'
```

---

No, it's actually manual